.github/workflows/assign-reviewers.yml

```diff
@@ -0,0 +1,21 @@
+name: Assign reviewers
```

Comment:
Nit: maybe call this something like "Automate assignees", i.e., remove "reviewers" from the name to avoid confusion?

Reply:
Renamed to "Choose random assignees"